CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# rapids-cmake 22.06.00 (Date TBD)
+
+Please see https://github.com/rapidsai/rapids-cmake/releases/tag/v22.06.00a for the latest changes to this development branch.
+
 # rapids-cmake 22.04.00 (Date TBD)
```

Comment:
It looks like the changelog hasn't been updated for 21.12, 22.02, or 22.04. @robertmaynard @rapidsai/ops-codeowners Can someone investigate this?

Reply:
I need to defer to ops, as I don't know how the release process collects and updates the change log